easybuild/easyblocks/p/pgi.py

```diff
@@ -144,11 +151,14 @@ def install_step(self):
                 if os.path.islink(path):
                     os.remove(path)
 
-        # install (or update) siterc file to make PGI consider $LIBRARY_PATH
+        # install (or update) siterc file to make PGI consider $LIBRARY_PATH and accept -pthread
         siterc_path = os.path.join(self.installdir, self.pgi_install_subdir, 'bin', 'siterc')
         write_file(siterc_path, SITERC_LIBRARY_PATH, append=True)
         self.log.info("Appended instructions to pick up $LIBRARY_PATH to siterc file at %s: %s",
                       siterc_path, SITERC_LIBRARY_PATH)
+        write_file(siterc_path, SITERC_PTHREAD_SWITCH, append=True)
+        self.log.info("Appended instructions to allow -pthread switch to siterc file at %s: %s",
```

Comment:
And "Append instructions to replace -pthread with -lpthread to siterc file at ..." feels better to me.